canonical-web-and-design/juju.is
# Build the about model-driven-operations page

## QA
- Go to http://0.0.0.0:8041/about/model-driven-operations
- Check it matches the [copy doc](https://docs.google.com/document/d/1_NL5GxZgsP6OURWepwBSIuZOKmEzmwaAqnnaRhfJ4lU/edit)

Fixes https://github.com/canonical-web-and-design/juju.is/issues/98
